RecoTracker/LSTCore/src/alpaka/Event.h

```diff
@@ -87,9 +87,9 @@ namespace lst {
     const uint16_t nLowerModules_;
     const unsigned int nPixels_;
     const unsigned int nEndCapMap_;
-    const std::shared_ptr<const ModulesBuffer<Device>> modulesBuffers_;
-    const std::shared_ptr<const PixelMap> pixelMapping_;
-    const std::shared_ptr<const EndcapGeometryBuffer<Device>> endcapGeometryBuffers_;
+    ModulesBuffer<Device> const& modulesBuffers_;
+    std::shared_ptr<const PixelMap> pixelMapping_;
```

Comment:
why not a  `PixelMap const& pixelMapping_;` here as well?

Reply:
Yeah, I'll make it more consistent